src/MaRRTPathPlanNode.py

```diff
@@ -45,14 +42,24 @@ def __init__(self):
         if rospy.has_param('~filename'):
             self.filename = rospy.get_param('~filename')
 
+        if rospy.has_param('~odom_topic'):
+            self.odometry_topic = rospy.get_param('~odom_topic')
+        else:
+            self.odometry_topic = "/odometry"
+
+        if rospy.has_param('~world_frame'):
+            self.world_frame = rospy.get_param('~world_frame')
+        else:
+            self.world_frame = "world"
+
         waypointsFrequency = rospy.get_param('~desiredWaypointsFrequency', 5)
         self.waypointsPublishInterval = 1.0 / waypointsFrequency
         self.lastPublishWaypointsTime = 0
 
         # All Subs and pubs
-        rospy.Subscriber("/map", Map, self.mapCallback)
-        rospy.Subscriber("/odometry", Odometry, self.odometryCallback)
-        # rospy.Subscriber("/car_sensors", CarSensors, self.carSensorsCallback)
+        rospy.Subscriber("/track", Track, self.mapCallback)
+        rospy.Subscriber(self.odometry_topic, Odometry, self.odometryCallback)
+        rospy.Subscriber("/dvsim/cmd", Command, self.carSensorsCallback)
```

Comment:
I don't really use the steering angle of the car for path planning (this subscriber was even commented out, but I tried to use this information during my development).
But in your version it appears to be confusing: you subscribe to a topic (which generally means to get something), and you receive _a command_ (which generally would be understood as control request).
Maybe message should be named a bit differently, what about CarState?